src/commands/force/lightning/lwc/test/run.ts

```diff
@@ -54,10 +55,11 @@ export default class Run extends SfdxCommand {
 
     const scriptRet = this.runJest(args);
 
-    this.ux.log(messages.getMessage('logSuccess', [scriptRet.status.toString()]));
+    const status = scriptRet.status ?? -1
```

Comment:
This is the only functional change I made. Seems sensible if the `status` is null to make the return integer -1.